src/einsteinpy/radiative_transfer.py

```diff
@@ -0,0 +1,56 @@
+import astropy.units as u
+import numpy as np
+
+from einsteinpy import constant
+from einsteinpy.coordinates import SphericalConversion
+from einsteinpy.integrators import RK45
+
+'''create a class photon which would include a position vector and a velocity vector'''
+class radiative_transfer:
+    '''This class holds methods to compute pitch angle , emmission coeffecient and radiative integration '''
+    def __init__(self,photons,metric): 
+        self.photon_pos=photon.position_vector #list of postions of individual photons
```

Comment:
Where is `photon` defined?

Reply:
As a separate class as of now, but I assume it can be put inside a related parent class i am unsure depending on whether we need this ?